oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -85,7 +85,8 @@ mca_spml_ucx_t mca_spml_ucx = {
 mca_spml_ucx_ctx_t mca_spml_ucx_ctx_default = {
     .ucp_worker = NULL,
     .ucp_peers  = NULL,
-    .options    = 0
+    .options    = 0,
+    .spml_ucx   = &mca_spml_ucx
```

Comment:
maybe just move synchronized_quiet to this struct? instead of moving &mca_spml_ucx

Reply:
in this case we have to additionally handle initialization of synchronized_quiet and update this value in global object - it makes fix more complicated

Reply:
i think it will worth it: better to keep "mca_spml_ucx" private, and mca_spml_ucx_ctx_t "shared"
and not point from "shared" to "private"

Reply:
ok, then may be pointer to synchronized_quiet?

Reply:
i think better just move this bool field. no need to keep it in 2 places.

Reply:
ok, done